ui/core/components/detailed_results/threat_metrics.tsx

```diff
@@ -0,0 +1,335 @@
+				},
+			},
+			{
+				name: 'Hits',
+				getValue: (metric: ActionMetrics) => metric.landedHits || metric.landedTicks,
+				fillCell: (metric: ActionMetrics, cellElem: HTMLElement) => {
+					cellElem.appendChild(
+						<>
+							{formatToNumber(metric.landedHits || metric.landedTicks, { fallbackString: '-' })}
+							{metric.landedHits && metric.landedTicks ? <> ({formatToNumber(metric.landedTicks, { fallbackString: '-' })})</> : undefined}
+						</>,
+					);
+					if (!metric.landedHits && !metric.landedTicks) return;
+
+					const relativeHitPercent = ((metric.hits - metric.resistedHits) / metric.landedHits) * 100;
+					const relativeResistedHitPercent = (metric.resistedHits / metric.landedHits) * 100;
+					const relativeCritPercent = ((metric.crits - metric.resistedCrits) / metric.landedHits) * 100;
+					const relativeResistedCritPercent = (metric.resistedCrits / metric.landedHits) * 100;
+					const relativeTickPercent = ((metric.ticks - metric.resistedTicks) / metric.landedTicks) * 100;
+					const relativeResistedTickPercent = (metric.resistedTicks / metric.landedTicks) * 100;
+					const relativeCritTickPercent = ((metric.critTicks - metric.resistedCritTicks) / metric.landedTicks) * 100;
+					const relativeResistedCritTickPercent = (metric.resistedCritTicks / metric.landedTicks) * 100;
+					const relativeGlancePercent = (metric.glances / metric.landedHits) * 100;
+					const relativeBlockPercent = (metric.blocks / metric.landedHits) * 100;
+					const relativeCritBlockPercent = (metric.critBlocks / metric.landedHits) * 100;
+
+					cellElem.appendChild(
+						<MetricsCombinedTooltipTable
+							tooltipElement={cellElem}
+							groups={[
+								{
+									spellSchool: metric.spellSchool,
+									total: metric.landedHits,
+									totalPercentage: 100,
+									data: [
+										{
+											name: 'Hit',
+											value: metric.hits - metric.resistedHits,
+											percentage: relativeHitPercent,
+										},
+										{
+											name: 'Resisted Hit',
+											value: metric.resistedHits,
+											percentage: relativeResistedHitPercent,
+										},
+										{
+											name: `Critical Hit`,
+											value: metric.crits - metric.resistedCrits,
+											percentage: relativeCritPercent,
+										},
+										{
+											name: `Resisted Critical Hit`,
+											value: metric.resistedCrits,
+											percentage: relativeResistedCritPercent,
+										},
+										{
+											name: 'Glancing Blow',
+											value: metric.glances,
+											percentage: relativeGlancePercent,
+										},
+										{
+											name: 'Blocked Hit',
+											value: metric.blocks,
+											percentage: relativeBlockPercent,
+										},
+										{
+											name: 'Blocked Critical Hit',
+											value: metric.critBlocks,
+											percentage: relativeCritBlockPercent,
+										},
```

Comment:
Blocked crits don't exist in SoD 😄 removed in a commit earlier

Reply:
I didn't know :D Done!